packages/video_player_avplay/lib/video_player.dart

```diff
@@ -973,6 +992,15 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     return _videoPlayerPlatform.getData(playerId, keys);
   }
 
+  /// Update token of DASH at any time after initialization is complete.
+  Future<bool> updateToken(String token) async {
+    if (_isDisposedOrNotInitialized) {
+      return false;
+    }
+
+    return _videoPlayerPlatform.updateToken(playerId, token);
+  }
```

Comment:
When I set UpdateDashToken() when the current resource type or format is not "dash" the following error will continue to occur through listener.
Since PlusPlayer's API explicitly states that it's Dash-only(`::UpdateDashToken(player_, token);`),
I think the interface should also indicate that it's a Dash-only API.
(And if it's not a dash, it can output related exception or error messages.)

```
[E] [ERROR:flutter/runtime/dart_vm_initializer.cc(40)] Unhandled Exception: PlatformException(channel-error, Unable to establish connection on channel: "dev.flutter.pigeon.video_player_avplay.VideoPlayerAvplayApi.updateToken"., null, null)
#0      VideoPlayerAvplayApi.updateToken (package:video_player_avplay/src/messages.g.dart:1782:7)
<asynchronous suspension>
```